website/blog/2021-05-20-GAAD-One-Year-Later.md

```diff
@@ -0,0 +1,119 @@
+---
+title: The GAAD Pledge - One Year Later
+author: Alexandra Marlette
+authorTitle: GAAD Pledge Open Source Accessibility Community Manager for React Native
+authorURL: 'https://twitter.com/alexmarlette'
+authorImageURL: 'https://avatars.githubusercontent.com/u/10052470?s=460&u=7f2304cb929d1de703856717af86324c66728f3a&v=4'
+authorTwitter: alexmarlette
+tags: [announcement]
+---
+
+It has been one year since Facebook took the [GAAD Pledge](https://diamond.la/GAADPledge/) to make React Native accessible and the project has exceeded our expectations. We are excited to announce that this project will continue throughout 2021 and want to update everyone on our progress so far. Following a thorough analysis of the accessibility gaps in React Native last year, work began on filling these gaps.
+
+When we started there were 90 issues identified, since March when the gap analysis issues launched on Github:
```

Comment:
Suggestion: 
> We started with 90 outstanding gap analysis issues and from March 2021, when the project launched on Github, until now:

Reply:
I like that wording, I'll replace the current wording.